ERR Unsupported CONFIG parameter: notify-keyspace-events

I try to run the samples, but I always get the exception below. How do I fix the problem? Do I need to change some versions?

```
Caused by: org.springframework.dao.InvalidDataAccessApiUsageException: ERR Unsupported CONFIG parameter: notify-keyspace-events; nested exception is redis.clien
ts.jedis.exceptions.JedisDataException: ERR Unsupported CONFIG parameter: notify-keyspace-events
        at org.springframework.data.redis.connection.jedis.JedisExceptionConverter.convert(JedisExceptionConverter.java:44)
        at org.springframework.data.redis.connection.jedis.JedisExceptionConverter.convert(JedisExceptionConverter.java:36)
        at org.springframework.data.redis.PassThroughExceptionTranslationStrategy.translate(PassThroughExceptionTranslationStrategy.java:37)
        at org.springframework.data.redis.FallbackExceptionTranslationStrategy.translate(FallbackExceptionTranslationStrategy.java:37)
        at org.springframework.data.redis.connection.jedis.JedisConnection.convertJedisAccessException(JedisConnection.java:195)
        at org.springframework.data.redis.connection.jedis.JedisConnection.setConfig(JedisConnection.java:618)
        at org.springframework.session.data.redis.config.annotation.web.http.RedisHttpSessionConfiguration$EnableRedisKeyspaceNotificationsInitializer.afterProp
ertiesSet(RedisHttpSessionConfiguration.java:170)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1625)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1562)
        ... 22 more
Caused by: redis.clients.jedis.exceptions.JedisDataException: ERR Unsupported CONFIG parameter: notify-keyspace-events
        at redis.clients.jedis.Protocol.processError(Protocol.java:113)
        at redis.clients.jedis.Protocol.process(Protocol.java:138)
        at redis.clients.jedis.Protocol.read(Protocol.java:192)
        at redis.clients.jedis.Connection.readProtocolWithCheckingBroken(Connection.java:282)
        at redis.clients.jedis.Connection.getStatusCodeReply(Connection.java:181)
        at redis.clients.jedis.Jedis.configSet(Jedis.java:2480)
        at org.springframework.data.redis.connection.jedis.JedisConnection.setConfig(JedisConnection.java:616)
        ... 25 more
```

--- a/spring-session/build.gradle
+++ b/spring-session/build.gradle
@@ -22,7 +22,7 @@ dependencies {
     provided "javax.servlet:javax.servlet-api:$servletApiVersion"
     integrationTestCompile "redis.clients:jedis:2.4.1",
             "org.apache.commons:commons-pool2:2.2",
-            "redis.embedded:embedded-redis:0.2"
+            "redis.embedded:embedded-redis:0.4"
     testCompile 'junit:junit:4.11',
             'org.mockito:mockito-core:1.9.5',
             "org.springframework:spring-test:$springVersion",

